src/Features/Core/Portable/IntroduceVariable/AbstractIntroduceVariableService.cs

```diff
@@ -386,22 +386,23 @@ bool IsInstanceMemberReference(IOperation operation)
             if (typeInfo.Type?.SpecialType == SpecialType.System_String &&
                 typeInfo.ConvertedType?.IsFormattableString() == true)
             {
-                return typeInfo.ConvertedType;
+                return typeInfo.ConvertedType.WithNullability(typeInfo.Nullability.FlowState);
```

Comment:
There's also ConvertedNullability.FlowState; this should probably be using that.

Reply:
@ryzngard Did you meant to update this one too? There's two uses here.

Reply:
hrmm... an extension would be useful here.  it would have likely prevented this.

Reply:
Yea, I'll add an extension for this. `typeInfo.GetConvertedTypeWithNullability()` and `typeInfo.GetTypeWithNullability()` seem reasonable